integration-tests/src/tests/definition.test.ts

```diff
@@ -86,7 +90,7 @@ suite('Bitbake Definition Test Suite', () => {
   test('Definition appears properly on Bash variable', async () => {
     const position = new vscode.Position(15, 3)
     const expectedPathEnding = filePath
-    const expectedRange = new vscode.Range(13, 2, 13, 8)
+    const expectedRange = new vscode.Range(14, 2, 14, 8)
```

Comment:
Not sure why this would fail only "randomly" two months later. Looks like something that should have been failing all the time.

Reply:
This is not the part that failed randomly. The second commit is about checkIsRangeEqual not being asserted so this test was wrong all the time but undetected.
it's the first commit that seems to fix the randomness (i'm launching the CI a few times to confirm). The type Location[]/LinkLocation[] was not handled properly

Reply:
LocationLink is an improved "Location" if any extension or ourselves started emitting one, then I guess the array chaged type.